.env.example

```
@@ -0,0 +1 @@
+REACT_GIFT_MOCK_SERVER_ENDPOINT=localhost:3000
```

라이브 강의에서 민감한 정보는 환경 변수로 관리하라고 하셔서 연습해볼겸 백엔드 주소를 환경 변수로 넣었습니다. .env 파일은 업로드 되지 않았고 프로젝트 세팅할때 참고하라고 .env.example를 만들었습니다.

---

참고를 위한 좋은 가이드 파일을 만드셨네요! 소소하지만 앞에 `VITE_`가 추가되어야 할 것 같습니다 ㅎㅎ

```suggestion
VITE_REACT_GIFT_MOCK_SERVER_ENDPOINT=localhost:3000
```